specifications/api/swagger.yaml

```diff
@@ -1363,6 +1410,21 @@ paths:
         - *parametersWalletId
       responses: *responsesGetDelegationFee
 
+  /wallets/{walletId}/coin-selections/random:
+    post:
+      operationId: selectCoins
+      tags: ["Coin Selections"]
+      summary: Random
+      description: |
+        <p align="right">status: <strong>not implemented</strong></p>
+
+        Select coins to cover the given set of payments.
```

Comment:
You could add "... using the [Random-Improve coin selection algorithm](https://iohk.io/blog/self-organisation-in-coin-selection/)" as an explanation for what the path part "random" alludes to.

Reply:
> You could add "... using the [Random-Improve coin selection algorithm](https://iohk.io/blog/self-organisation-in-coin-selection/)" as an explanation for what the path part "random" alludes to.

Done. See: https://github.com/input-output-hk/cardano-wallet/pull/1152/commits/2e7e18b1080b6b9a55002af0e7fd0f7d7fefcf3a